MH4U Dex/CoreDataController.m

```diff
@@ -130,7 +130,7 @@ - (void)loadMonsterDropData
     for (NSDictionary *drop in dropList) {
         // Only add the drop if it is not already there
         NSNumber *monsterDropID = drop[MonsterDropIDKey];
-        NSPredicate *monsterDropPredicate = [NSPredicate predicateWithFormat:@"%K == %d", MonsterDropIDKey, monsterDropID.intValue];
+        NSPredicate *monsterDropPredicate = [NSPredicate predicateWithFormat:@"%K == %d", MonsterAttributes.id, monsterDropID.intValue];
```

Comment:
With this (and the similar changes below), are there any constants at the top of this file that aren't used at all anymore?